frontend/src/assets/images/small-pin.svg

```diff
@@ -1,6 +1,8 @@
-<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
-<ellipse cx="20.0004" cy="6.74412" rx="1.53846" ry="1.62791" fill="#F1F1F2"/>
-<ellipse cx="20.8002" cy="5.9575" rx="1.6" ry="1.70213" fill="#F1F1F2"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M25.7054 10.9374C26.5225 9.86298 27 8.48487 27 6.82927C27 3.05757 23.6421 0 19.5 0C15.3579 0 12 3.05757 12 6.82927C12 8.48705 12.4788 9.86662 13.2978 10.9417L19.4843 20L25.7054 10.9374Z" fill="#795F48"/>
-<circle cx="19.5" cy="7.5" r="2.5" fill="#F1F1F2"/>
-</svg>
+<svg width="70" height="70" viewBox="0 0 70 70" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What's the change to the SVG? Was this approved by Tatianna?

Reply:
He just removed the pin that was saved in `/assets`, but never used, and replaced it with the pin that was saved in `/public` which was the current pin in use